### PR TITLE
Removed non used code

### PR DESCRIPTION
**Pull request by danielcsant**
#### Description
Removed non used code

#### Reviewer
@gserranojc 

#### Test
All passed

#### Scalastyle
All test passed.

### CODE REVIEW

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala [hunk 1]
```diff
@@ -68,18 +66,12 @@ class CassandraOutput(keyName: String,
 
   override val clusteringPrecisions = properties.getString("clusteringPrecisions", None).map(_.split(fieldsSeparator))
 
-  override val indexFields = properties.getString("indexFields", None).map(_.split(fieldsSeparator))
```
Comment (gserranojc) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: Is used

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala [hunk 2]
```diff
@@ -68,18 +66,12 @@ class CassandraOutput(keyName: String,
...
-
   override val textIndexFields = properties.getString("textIndexFields", None).map(_.split(fieldsSeparator))
 
   override val analyzer = properties.getString("analyzer", DefaultAnalyzer)
 
   override val dateFormat = properties.getString("dateFormat", DefaultDateFormat)
 
-  override val refreshSeconds = properties.getString("refreshSeconds", DefaultRefreshSeconds)
```
Comment (gserranojc) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: is Used in Cassandra Stratio for lucene indexes.

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala [hunk 3]
```diff
@@ -68,18 +66,12 @@ class CassandraOutput(keyName: String,
...
-
-  override val textIndexName = properties.getString("textIndexName", "lucene")
```
Comment (gserranojc) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: is Used in Cassandra Stratio for lucene indexes.

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala [hunk 4]
```diff
@@ -99,19 +91,6 @@ class CassandraOutput(keyName: String,
     bcSchema.exists(bc => createTables(schemaFiltered, timeName, isAutoCalculateId))
   } else false
 
-  val indexesCreated = if (keyspaceCreated && tablesCreated && indexFields.isDefined && !indexFields.get.isEmpty) {
```
Comment (gserranojc) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: You can omit the result in a val, but the creation is necessary.

If you remove the val, you must check if the indexes are created when run sparkta.

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/dao/CassandraDAO.scala [hunk 5]
```diff
@@ -28,16 +28,11 @@ trait CassandraDAO extends Closeable with Logging {
 
   val DefaultAnalyzer = "english"
   val DefaultDateFormat = "yyyy/MM/dd"
-  val DefaultRefreshSeconds = "1"
```
Comment (gserranojc) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/dao/CassandraDAO.scala]: is used

**Comment by stratiocommit** [comment]
Jenkins job pending... wait for confirmation message before merging.

**Comment by stratiocommit** [comment]
Jenkins job went OK. Its up to someone to merge.

**Pull request closed without merging**